fv3config/fv3run/_native.py

```diff
@@ -50,11 +51,22 @@ def run_native(config_dict_or_location, outdir, runfile=None):
             _run_experiment(
                 localdir,
                 n_processes,
-                runfile_name=_get_basename_or_none(runfile),
+                python_args=_get_python_args(runfile),
```

Comment:
Also, could the `_get_executable_args` call be moved to just inside `_run_experiment`? Here you would just pass `runfile` to `runfile_name`. The logic for constructing argument lists is generally inside `_run_experiment`, while `run_native` mostly only contains filesystem/pipe management.

Reply:
done.